frontend/src/app/services/fire-service/fire.service.ts

```diff
@@ -14,11 +16,14 @@ export class FireService {
     }
 
     searchFirePolygon(id, size): Observable<object> {
-        return this.http.post('http://127.0.0.1:5000/data/fire-with-id', JSON.stringify({id, size}));
+        return this.http.post('http://' + environment.host + ':' + environment.port + '/data/fire-with-id', JSON.stringify({
+            id,
+            size
+        }));
     }
 
     searchSeparatedFirePolygon(id, size): Observable<object> {
-        return this.http.post('http://127.0.0.1:5000/data/fire-with-id-seperated', JSON.stringify({
+        return this.http.post('http://${environment.host}:${environment.port}/data/fire-with-id-seperated', JSON.stringify({
```

Comment:
please use all this format, instead of concatenating strings

Reply:
Sorry the one with the '$' is actually the one I forgot to change after the test, because this kind of format doesn't work. I had to use the concatenated ones.